### ISSUE TITLE
Import -> paste raw text -> X-XSRF-TOKEN with extra empty space

### ISSUE THREAD

**Issue by egor-xyz** [a85/POSTMan-Chrome-Extension#8268]
**Describe the bug**
When you are importing from Chrome and Import with **paste raw text** `X-XSRF-TOKEN` looks like this: `X-X SRF-TOKEN` and it does not work (space between X and S)

**To Reproduce**
Steps to reproduce the behaviour:
1. Open `Import` and choose tab: `paste raw text`
2. Paste 
```
curl 'http://localhost:3000/test' -H 'X-XSRF-TOKEN: test=' --compressed
```
3. Header parsed with extra space `X-X SRF-TOKEN`

**Expected behaviour**
The header must parse correct `X-XSRF-TOKEN`

**Screenshots**
![image](https://user-images.githubusercontent.com/6418221/78126291-febd1e00-741a-11ea-9f30-fb9af5bf1481.png)

**App information (please complete the following information):**
 - App Type Native App
 - Postman Version 7.21.1
 - OS: macOS Mojave 10.14.6

**Comment by adrianDY** [comment]
The same thing is happening to me since the upgrade to Version 7.21.1

**Comment by shreys7** [comment]
@egor-xyz @adrianDY thanks for reporting this issue. It is a known issue and we have added the fix for the same. It will be released with the next Postman App version - **7.22**, planned on 8th April.

**Comment by shreys7** [comment]
Duplicate of #7806
